<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-E13KQM6" ignored="0" name="Test" tests="5" failures="3" timestamp="2022-04-07T09:43:21 IST" time="174.781" errors="0">
  <testcase name="loginDDT" time="94.418" classname="com.fis.testCases.TC_LoginDTT_002"/>
  <testcase name="loginDDT" time="10.985" classname="com.fis.testCases.TC_LoginDTT_002">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.fis.testCases.TC_LoginDTT_002.loginDDT(TC_LoginDTT_002.java:32)
at java.util.ArrayList.forEach(ArrayList.java:1257)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- loginDDT -->
  <testcase name="loginDDT" time="12.94" classname="com.fis.testCases.TC_LoginDTT_002"/>
  <testcase name="loginDDT" time="26.761" classname="com.fis.testCases.TC_LoginDTT_002">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.fis.testCases.TC_LoginDTT_002.loginDDT(TC_LoginDTT_002.java:32)
at java.util.ArrayList.forEach(ArrayList.java:1257)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- loginDDT -->
  <testcase name="loginDDT" time="13.354" classname="com.fis.testCases.TC_LoginDTT_002">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.fis.testCases.TC_LoginDTT_002.loginDDT(TC_LoginDTT_002.java:32)
at java.util.ArrayList.forEach(ArrayList.java:1257)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- loginDDT -->
</testsuite> <!-- Test -->
